Gems/ROS2/repo.json

```diff
@@ -0,0 +1,11 @@
+{
+    "repo_name":"o3de-ros2-gem",
+    "origin":"RobotecAI",
+    "repo_uri": "https://github.com/RobotecAI/o3de-ros2-gem",
+    "summary": "A Gem Repository for ROS2 support",
+    "additional_info": "",
+    "last_updated": "",
+    "gems": [
+        "https://github.com/RobotecAI/o3de-ros2-gem"
```

Comment:
FYI: This will need to be updated for the final submission to point to the o3de-extras repo.

Reply:
@michalpelka please update

Reply:
Removed this file, done.